test/unit/workers/document_list_export_worker_test.rb

```diff
@@ -4,6 +4,25 @@ class DocumentListExportWorkerTest < ActiveSupport::TestCase
   setup do
     @user = create(:user)
     @worker = DocumentListExportWorker.new
+
+    Fog.mock!
+    ENV["AWS_REGION"] = "eu-west-1"
+    ENV["AWS_ACCESS_KEY_ID"] = "test"
+    ENV["AWS_SECRET_ACCESS_KEY"] = "test"
+    ENV["AWS_S3_BUCKET_NAME"] = "test-bucket"
+
+    # Create an S3 bucket so the code being tested can find it
+    connection = Fog::Storage.new(
+      provider: "AWS",
+      region: ENV["AWS_REGION"],
+      aws_access_key_id: ENV["AWS_ACCESS_KEY_ID"],
+      aws_secret_access_key: ENV["AWS_SECRET_ACCESS_KEY"],
+    )
+    @directory = connection.directories.get(ENV["AWS_S3_BUCKET_NAME"]) || connection.directories.create(key: ENV["AWS_S3_BUCKET_NAME"]) # rubocop:disable Rails/SaveBang
```

Comment:
Is it worth extracting this setup stuff into a helper?

EDIT - I see this is also copied in the third commit, so three times total? Definitely worth moving into a helper IMO 👍